hawkbit-rest/hawkbit-rest-core/src/test/java/org/eclipse/hawkbit/rest/AbstractRestIntegrationTest.java

```diff
@@ -51,6 +55,7 @@ public void before() throws Exception {
     protected DefaultMockMvcBuilder createMvcWebAppContext(final WebApplicationContext context) {
         final DefaultMockMvcBuilder createMvcWebAppContext = MockMvcBuilders.webAppContextSetup(context);
 
+        createMvcWebAppContext.addFilter(characterEncodingFilter);
```

Comment:
A comment explaining why this is required here wood be of help I think

Reply:
Comment has been added via the last commit.